viz.yaml

```diff
@@ -359,12 +363,12 @@ publish:
     template: fullpage
     scripts: scripts/publish/json.R
     depends: ["footer","footerCSS","header-section", "figure-section", "data-sources", "map-figure-css", 
-      "header-footer-css", "precip-colors", "timesteps_pub", "jquery-js", "animate-js", "timesteps",
+      "header-footer-css", "precip-colors", "timesteps_pub", "jquery-js", "animate-js", "vizlab-js", "timesteps",
```

Comment:
I was just helping Laura and I'm pretty sure this is going to conflict, but :+1: